r-pkg/uptasticsearch/tests/testthat/test-repo_characteristics.R

```diff
@@ -39,7 +39,7 @@ test_that("R CMD check should not return any errors", {
 })
 
 test_that("R CMD check should not return any warnings", {
-    expect_true(length(x[["warnings"]]) == 0)
+    expect_true(length(x[["warnings"]]) <= 1)
```

Comment:
I think this is unnecessary, and it should be `== 0`. On a system without `pdflatex`, it flags a warning in its verbosity, but does not store the WARNING in the final result of R CMD check (i.e. `x[["warnings"]]` should be an empty character). 

If that's not true on someone else's system, please let me know. And if that's the case, we should make this test something like `expect_true(length(x[["warnings"]]) == 0 || x[["warnings"]] == "warning you don't have latex ya noob")`.